jkube-kit/doc/src/main/asciidoc/inc/enricher/dependency/_jkube_dependency.adoc

```diff
@@ -0,0 +1,28 @@
+
+[[jkube-dependency]]
+==== jkube-dependency
+
+This enricher is used for embedding dependency descriptors to single package. It looks for the following files in compile scope dependencies and adds {cluster} resources inside to final generated {cluster} manifests:
```

Comment:
I know `dependency descriptor` is specifically mentioned in the DependencyEnricher Javadoc.

Do you think that this name is accurate?

I understand that something like `cluster configuration manifests (YAML)` or something in those lines might be a more accurate or user friendly description for what this enricher is "embedding".